chalicelib/blueprints/announcements.py

```diff
@@ -0,0 +1,55 @@
+from chalice import Blueprint
+from chalicelib.services.ses import SesMailSender, SesDestination
+from chalicelib.services.mongo import MongoService
+import boto3
+
+announcements_routes = Blueprint(__name__)
+
+ses_client = boto3.client('ses')
+
+
+# why is GET needed?
+@announcements_routes.route("/announcement", methods=["GET", "POST"], cors=True)
```

Comment:
I think `GET` isn't needed in this case – just have `POST`!

Reply:
For some reason, GET requests are still made.
![image](https://github.com/whyphi/zap/assets/57511176/4d74d598-600e-43b1-9a59-900cc0a8706e)
Adding the method works as a temp fix- I will deal with this later as it doesn't seem like a high priority issue.